tests/infra_test.go

```diff
@@ -303,9 +301,22 @@ var _ = Describe("Infrastructure", func() {
 			keys := getKeysFromMetrics(metrics)
 			for _, key := range keys {
 				// we don't care about the ordering of the labels
-				Expect(key).To(ContainSubstring(fmt.Sprintf("node=%s", nodeName)))
-				Expect(key).To(ContainSubstring(fmt.Sprintf("namespace=%s", vmi.Namespace)))
-				Expect(key).To(ContainSubstring(fmt.Sprintf("name=%s", vmi.Name)))
+				Expect(key).To(ContainSubstring(fmt.Sprintf("node=\"%s\"", nodeName)))
+				Expect(key).To(ContainSubstring(fmt.Sprintf("namespace=\"%s\"", vmi.Namespace)))
+				Expect(key).To(ContainSubstring(fmt.Sprintf("name=\"%s\"", vmi.Name)))
+			}
+		}, 300)
+
+		It("should include VMI telemetry for a running VM", func() {
+			_, _, _, metrics := newRandomVMIWithMetrics(virtClient, "kubevirt_vmi_")
```

Comment:
Do we now expose kubevirt_vm and kubevirt_vmi metrics?

Reply:
That's open for discussion. I don't have strong opinions here. What do you prefer?

Reply:
imo, if we talks about vmis, it should be about kubevirt_vmi. So if we had `kubevirt_vm` before, then I missed it on the reviews. ;)

Reply:
No problem renaming the metrics. Is this a breaking change compatibility-wise?

Reply:
Probably better changing it now where most likely no-one uses them yet. Can you ask on kubevirt-dev and #virtualization in the k8s slack? @fabiand I am concerned that we will create a mess on the long run if we are not exactly defining `VMI` related metrics with `vmi`.

Reply:
Filed https://github.com/kubevirt/kubevirt/issues/2504 to track this issue

Reply:
+1 on doing all metric renames that are necessary.

Reply:
Discussion started on kubevirt-dev and on slack. Posted trivial https://github.com/kubevirt/kubevirt/pull/2506